### PR TITLE
[Feat] - Changed Math.Sqrt Methods

### PR DESCRIPTION
**Pull request by adithyaakrishna**
**Description:**

This PR changes,

  - `Math.sqrt(dx * dx + dy * dy)` to `Math.hypot(dx, dy)`
  - `Math.sqrt(dy * dy + dz * dz)` to `Math.hypot(dy, dz)`

### CODE REVIEW

**Comment by adithyaakrishna** [comment]
But why `hypot()` compared to `sqrt()`?

**Benchmark**: https://measurethat.net/Benchmarks/Show/25689/0/hypot-vs-sqrt

**Comment by adithyaakrishna** [comment]
Gonna keep this PR sweet and small, @dkoes If it looks good, can we merge this?

File: src/GLViewer.ts [hunk 1]
```diff
@@ -2784,8 +2784,8 @@ export class GLViewer {
                 var u, v, w;
 
                 u = Math.cos(beta);
-                v = (Math.cos(alpha) - Math.cos(beta) * Math.cos(gamma)) / Math.sin(gamma);
-                w = Math.sqrt(Math.max(0, 1 - u * u - v * v));
```
Comment (dkoes) [inline src/GLViewer.ts:2788]: why is this changed?  It doesn't calculate the same value and removes the guard against generating a NaN.

**Review by dkoes — CHANGES_REQUESTED** [review]
Only hypot changes please.

File: src/GLViewer.ts (hunk 1 above)
Comment (adithyaakrishna) [reply src/GLViewer.ts:2788]: @dkoes Does u,v accept negative values?

Comment (dkoes) [reply src/GLViewer.ts:2788]: Why does that matter?

**Review by dkoes — APPROVED** [review]
